### DISCUSSION TITLE
Is there an equivalent to getStaticpath to next.

### DISCUSSION THREAD

**Discussion by thapecroth** [preactjs/wmr#598] [Q&A] [answered]
Hi, I am new here and have use nextjs before. I checked out the preact-router. I understand how I can do dynamic path. But still curious, about how it will know which page slug to prerender. Is there a way to do similar thing to next's getstaticpath?

**Comment by JoviDeCroock** [comment, marked as answer]
Paths should be automatically picked up by rendering links for them, let's take the example of our own documentation page, by rendering the links in the sidebar `wmr` will detect pages (if they're within the same domain) and prerender them, this means that given `--prerender` flag we'll SSG every page we can find for you.

**Reply by thapecroth** [reply]
Thank you. This is very cool. If there is a link, then it's must be public. Thus, prerender !

**Reply by danielweck** [reply]
You may also configure WMR's `customRoutes` if there are no public / discoverable links in your components:
https://wmr.dev/docs/configuration#customroutes
